__tests__/resolve-config.test.ts

```diff
@@ -85,4 +85,24 @@ describe('Resolves Config', () => {
     process.env = {}
     await expect(resolveConfig.resolveConfig(mockReporter)).rejects.toThrowError()
   })
+
+  describe('parses boolean', () => {
+    test('but throws if incorrect value', async () => {
+      process.env = {
+        ...process.env,
+        INPUT_TUNNEL: 'not_a_bool',
+      }
+      await expect(resolveConfig.resolveConfig(mockReporter)).rejects.toThrow(
+        /Input does not meet YAML 1.2 \"Core Schema\" specification: tunnel/
+      )
+    })
+
+    test('if value meets YAML 1.2', async () => {
+      process.env = {
+        ...process.env,
+        INPUT_TUNNEL: 'True',
+      }
+      expect(await (await resolveConfig.resolveConfig(mockReporter)).tunnel).toBeTruthy()
```

Comment:
I'm a bit confused why we need 2 `await` here 🤔  Can it be simplified with something like this?

```suggestion
      expect((await resolveConfig.resolveConfig(mockReporter)).tunnel).toBe(true)
```